src/peergos/shared/user/fs/FileWrapper.java

```diff
@@ -109,13 +109,6 @@ public FileWrapper withVersion(Snapshot version) {
         return new FileWrapper(capTrie, pointer, linkPointer, entryWriter, ownername, version);
     }
 
-    @JsMethod
-    public CompletableFuture<FileWrapper> getLatest(NetworkAccess network) {
```

Comment:
aren't we going to need this in other features?

Reply:
For video capture - yes. But as it doesn't seem to work when the file is subsequently shared it seemed prudent to remove